backend/server-connection/src/main/proto/sonarlint.proto

```diff
@@ -75,6 +76,11 @@ message Flow {
   repeated Location location = 1;
 }
 
+message OverriddenImpact {
```

Comment:
Same here, you could just have `Impact`.
EDIT: in fact it's already there in the file, so you could just reuse the existing message